src/applications/vaos/tests/e2e/00.appointment-list.e2e.spec.js

```diff
@@ -1,42 +1,30 @@
-const E2eHelpers = require('../../../../platform/testing/e2e/helpers');
 const Timeouts = require('../../../../platform/testing/e2e/timeouts.js');
 const VAOSHelpers = require('./vaos-helpers');
 const Auth = require('../../../../platform/testing/e2e/auth');
 
-module.exports = E2eHelpers.createE2eTest(client => {
-  const token = Auth.getUserToken();
+module.exports = {
+  after: (client, done) => {
+    client.deleteCookies();
+    client.end();
+    done();
+  },
+  login: client => {
+    const token = Auth.getUserToken();
 
-  VAOSHelpers.initAppointmentListMock(token);
-  E2eHelpers.overrideVetsGovApi(client);
+    VAOSHelpers.initAppointmentListMock(token);
 
-  // init announcements
-  client.execute(() => {
-    // window.localStorage.clear();
-    window.localStorage.setItem(
-      'DISMISSED_ANNOUNCEMENTS',
-      '["welcome-to-new-vaos"]',
-    );
-  });
-
-  Auth.logIn(
-    token,
-    client,
-    '/health-care/schedule-view-va-appointments/appointments/',
-    3,
-    VAOSHelpers.getUserDataWithFacilities(),
-  )
-    .waitForElementVisible('#appointments-list', Timeouts.slow)
-    .axeCheck('.main');
-
-  client
-    .click('.vaos-appts__cancel-btn')
-    .waitForElementVisible('#cancelAppt', Timeouts.slow)
-    .axeCheck('.main')
-    .click('#cancelAppt .usa-button')
-    .waitForElementVisible('.usa-alert-success', Timeouts.normal)
-    .axeCheck('.main')
-    .click('#cancelAppt button')
-    .waitForElementNotPresent('#cancelAppt', Timeouts.normal);
-
-  client.end();
-});
+    Auth.logIn(
+      token,
+      client,
+      '/health-care/schedule-view-va-appointments/appointments/',
+      3,
+      VAOSHelpers.getUserDataWithFacilities(),
+    ).waitForElementVisible('#appointments-list', Timeouts.slow);
+  },
+  'Cancel appointment': client => {
+    VAOSHelpers.cancelAppointmentTest(client);
```

Comment:
What's the reason for cancelling an appointment before testing the "show more" link? I think those should show up for any request, if we have the data mocked.

Reply:
The cancel appointment test was originally there. The 'show more' test can be moved to its own module if you want.

Reply:
Oh, gotcha, I misread the diff. The show more test can stay in here.